package.json

```diff
@@ -11,28 +11,47 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build && tsc && node copy-types.js",
+    "build:css": "tailwindcss -m -i ./src/tailwind-entry.css -o ./dist/styles.css",
+    "preview": "vite preview",
+    "storybook": "concurrently \"pnpm storybook:css\" \"storybook dev -p 6006\"",
+    "storybook:css": "tailwindcss -w -i ./src/tailwind-entry.css -o ./src/index.css",
+    "build-storybook": "concurrently \"pnpm build-storybook:css\" \"storybook build\"",
+    "build-storybook:css": "tailwindcss -m -i ./src/tailwind-entry.css -o ./src/index.css",
```

Comment:
tailwind-entry.css isn't necessary. It's the same file as the style.css. Remove the tailwind-entry.css file and replace the scripts here with style.css